itest/rust/src/virtual_methods_test.rs

```diff
@@ -215,3 +236,12 @@ fn test_tree_enters_exits(test_context: &TestContext) {
     assert_eq!(obj.bind().tree_enters, 2);
     assert_eq!(obj.bind().tree_exits, 1);
 }
+
+#[itest]
+fn test_virtual_method_with_return(_test_context: &TestContext) {
+    let mut obj = Gd::<ReturnVirtualTest>::new_default();
+    let output = obj.bind().get_configuration_warnings();
+    assert!(output.contains("Hello".into()));
+    assert_eq!(output.len(), 1);
+    obj.bind_mut().queue_free();
```

Comment:
This line missing was the cause of the memory leak and, therefore, of the pipeline failing. Another way of solving the memory leak would be to add the `obj` node to the `test_context` which, I guess, gets cleaned at the end of the test. I went for `queue_free` because it looked more concise, but I do not know if there is a recommended practice here.